packages/backend-plugin-api/api-report.md

```diff
@@ -205,9 +205,15 @@ export type LifecycleServiceShutdownHook = {
 // @public (undocumented)
 export interface LoggerService {
   // (undocumented)
-  child(fields: { [name: string]: string }): LoggerService;
+  child(meta: LogMeta): LoggerService;
```

Comment:
Hm. I realize it feels odd to do `.child(new Error())`.

Should we have `type LogMeta = { [name: string]: any; }`, and `debug(message: string, meta?: LogMeta | Error): void;`?